commons/commands.py

```diff
@@ -518,6 +518,12 @@
                                 "jsonpath='{{.items[-1:].metadata.name}}'"
 KUBECTL_GET_RPM = "kubectl exec -it {} -c {} -- rpm -qa|grep -i {}"
 KUBECTL_SET_CONTEXT = "kubectl config set-context --current --namespace={}"
+KUBECTL_GET_TAINT_NODES = "kubectl get nodes -o custom-columns=" \
+                          "NAME:.metadata.name,TAINTS:.spec.taints --no-headers > {}"
+KUBECTL_GET_ALL = "kubectl get all -A >> {}"
```

Comment:
Do we need to take care of namespace here also.

Reply:
no